bittensor/subtensor.py

```diff
@@ -1541,7 +1537,6 @@ def serve(
         placeholder2: int = 0,
```

Comment:
Do we actually need these placeholder params? What is their use? If we need them in the future, couldn't we just add them at that time?

Reply:
It uses for making `compose_call` as a part of `call_params` starting from [here](https://github.com/opentensor/bittensor/blob/master/bittensor/extrinsics/serving.py#L67)